### PR TITLE
Event Bus & Websocket:

### PR DESCRIPTION
**Pull request by jdrossl**
- Replace EBus with Spring Events
- Add new websocket & security configuration

https://github.com/craftercms/craftercms/issues/5325

Depends on https://github.com/craftercms/craftercms/pull/5349

### CODE REVIEW

File: src/main/java/org/craftercms/studio/api/v2/event/Person.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    protected final String firstName;
+
+    protected final String lastName;
+
+    public Person(String username, String firstName, String lastName) {
+        this.username = username;
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/api/v2/event/Person.java:46]: There is more to `Person`, see https://app.swaggerhub.com/apis/craftercms/studio/4.0.0.26#/Person

File: src/main/java/org/craftercms/studio/api/v2/event/Person.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        return new Person(user.getUsername(), user.getFirstName(), user.getLastName());
+    }
+
+    protected final String username;
+
+    protected final String firstName;
+
+    protected final String lastName;
+
+    public Person(String username, String firstName, String lastName) {
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/api/v2/event/Person.java:42]: Should `Person` be a TO used everywhere? We have other usecases.

File: src/main/java/org/craftercms/studio/api/v2/event/content/ContentEvent.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+package org.craftercms.studio.api.v2.event.content;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.craftercms.studio.api.v2.event.BroadcastEvent;
+import org.craftercms.studio.api.v2.event.SiteAwareEvent;
+import org.springframework.security.core.Authentication;
+
+/**
+ * @author joseross
+ * @since
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/api/v2/event/content/ContentEvent.java]: Since?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/java/org/craftercms/studio/api/v2/event/Person.java (hunk 1 above)
Comment (jdrossl) [reply src/main/java/org/craftercms/studio/api/v2/event/Person.java:46]: I think this class doesn't exist in the backend yet

File: src/main/java/org/craftercms/studio/api/v2/event/Person.java (hunk 2 above)
Comment (jdrossl) [reply src/main/java/org/craftercms/studio/api/v2/event/Person.java:42]: Yes I added it here temporarily until the real implementation is added and can be used across all services

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**